src/ui-mapbox/common.ts

```diff
@@ -427,18 +427,24 @@ export interface ListOfflineRegionsOptions {
 
 // ------------------------------------------------------------
 
+export enum ControlPosition {
+    TOP_LEFT,
+    TOP_RIGHT,
+    BOTTOM_LEFT,
+    BOTTOM_RIGHT
+}
+
 /**
  * The options object passed into the show function.
  */
-
 export interface ShowOptions {
```

Comment:
I've replaced the use of `any` with this interface in a few places, I did need to loosen the types to make this work, for example `center` becoming `Partial`. I didn't quite understand the difference between options and settings, so let me know if there's a better approach.